test/runnable/sameenv.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+$DMD -m${MODEL} -of${OUTPUT_BASE}/printenv${EXE} ${EXTRA_FILES}/printenv.d
+${OUTPUT_BASE}/printenv${EXE} > ${OUTPUT_BASE}/envFromExe.txt
+
+$DMD -m${MODEL} -run ${EXTRA_FILES}/printenv.d > ${OUTPUT_BASE}/envFromRun.txt
+
+diff ${OUTPUT_BASE}/envFromExe.txt ${OUTPUT_BASE}/envFromRun.txt
```

Comment:
I always found adding `-p` more helpful when looking at errors. Maybe we should overwrite add an `alias diff = diff -p` to the "bash runtime environment"?

Reply:
Also maybe you want to compare it with the output from `env`?

Reply:
output from env might be different, it sorts differently I believe.

Reply:
Then normalize both by  `| sort`?

Reply:
I get different outputs from `env` and this program on windows.  It sorts a bit different.  In any case, we don't really care that the output exactly matches the output of `env`, we just want to make sure running the executable directly matches `dmd -run`.